scenarios/retail/README.md

```diff
@@ -1,6 +1,6 @@
 # Recommendation systems for Retail

```

Comment:
I think this README is too long and broader than retail. Some considerations are not specific to retail, but apply to many business scenarios in general. Maybe we could move them to a separate page? 
Also the technical aspects could go to another page. E.g. one business problem is cold start for new users / products, so it can be here (or in a general page), how you address cold start (e.g. by content  based methods) goes to a technical page.

Reply:
Yes agreed.

Reply:
> Maybe we could move them to a separate page?

initially, it was proposed to have a general page to have the common content, but on a subsequent meeting that idea was rejected to have everything related to business here and technical parts in the examples section. I don't have a strong position on this tbh, but I think it would be good to clearly decide what to do